### DISCUSSION TITLE
Revoir les issues du SE fightPet

### DISCUSSION THREAD

**Discussion by Ntalcme** [Crownicles/Crownicles#2714] [Nouvelles idées]
J'aimerais bien qu'on revoit les issues sur le SE fightPet, certaines sont trop difficiles, d'autres trop simples.
Certaines devraient, selon moi, prendre plus en compte le level du joueur.

### Faciles
- Fuir 🏃 
- Se protéger 🛡️ 
- Coup de poing 👊 

Solution proposée pour elles : Prendre en compte le level du joueur **OU** changer la valeur de base des constantes pour chacune des 3. (respectivement actuellement 15, 60 et 50)
-> Problème de la solution : À bas level ça pourrait rendre encore plus dur les issues.

### Difficiles
- Provoquer l'animal 😤 : Complétement désavantageux à level >= 100 si on a pas l'arme insultes car on se prend 0% de chance.
- Prier Dieu 🙏 : beaucoup trop lié aux items sacrés, 0% qui vient trop facilement.

Solution proposée : Juste tout revoir.

### Calculs à revoir selon moi
- Intimider la créature 💪 : passer le maximum du level de 90 -> 100 me semble correct.
- Faire le mort 💀,  Utiliser toutes ses forces 🔥et Concentrer l'énergie ⚡:  Pour les 2 premières faut être sur les portes de la mort limite pour que ça soit viable, la deuxième même quasiment full on se prend vite une grosse descente de probabilité.
- Faire appel à son familier 🐕 : j'aimerais bien rajouter une condition qui est la suivante `Si player.pet.rarity == 5 && player.badges?.includes(BadgeConstants.PET_TAMER)` alors 100% de chance.

Voilà voilà, je suis pas trop rentré dans les détails donc si vous vous demandez pourquoi, lisez le code ou posez moi la question.

**Comment by Procion06** [comment]
ce que tu proposes me semble correct
globalement, on ne gagne pas assez de rage durant les runs
elle devient pourtant vite cruciale à haut level
quand on tombe sur un mini event de rage, il arrive régulièrement que toutes les issues soient défavorables pour les raisons que tu viens de citer
il est quasi impossible de choper de la rage en faisant le mort ou en concentrant son énergie
et pour les items spécifiques utiles pour provoquer l'animal ou prier, l'idée en soi est bonne, mais passé un certain level personne ne garde de tels items tellement l'inventaire est limité
faut rester réaliste, on a besoin de garder des stats correctes, que ce soit pour les boss ou le pvp

on en a déjà discuté également, les mini events où il ne se passe rien sont trop fréquents et ce serait juste de diminuer leur fréquence
l'idée proposée par Drapht d'ajouter un point de repos pour récupérer son énergie serait vraiment utile, à défaut augmenter les gains d'énergie des minis events car à haut level ça devient vraiment ridicule
quitte à revoir les issues sur le SE fightPet, autant faire ça aussi :)

il faut que l'île reste difficile, mais plus on monte en level et plus ça devient un vrai parcours du combattant, de quoi dégoûter les joueurs, même les plus chevronnés
un équilibrage est donc le bienvenu
